deploy-as-code/helm/charts/core-services/egov-pdf/values.yaml

```diff
@@ -58,38 +58,25 @@ env: |
       configMapKeyRef:
         name: egov-service-host
         key: pdf-service
-  - name: EGOV_USER_HOST
-    valueFrom:
-      configMapKeyRef:
-        name: egov-service-host
-        key: egov-user
-  - name: EGOV_SEARCHER_HOST
+  - name: EGOV_PROJECT_HOST
     valueFrom:
       configMapKeyRef:
         name: egov-service-host
-        key: egov-searcher
-  - name: EGOV_PAYMENTS_HOST
-    valueFrom:
-      configMapKeyRef:
-        name: egov-service-host
-        key: collection-services
-  - name: EGOV_PTSERVICES_HOST
-    valueFrom:
-      configMapKeyRef:
-        name: egov-service-host
-        key: property-services   
-  - name: EGOV_TRADELICENSESERVICES_HOST
+        key: project-management-system     
+  - name: EGOV_USER_HOST
     valueFrom:
       configMapKeyRef:
         name: egov-service-host
-        key: tl-services 
+        key: egov-user  
   - name: EGOV_WORKFLOW_HOST
     valueFrom:
       configMapKeyRef:
         name: egov-service-host
         key: egov-workflow-v2   
   - name: CONTEXT_PATH
-    value: "/egov-pdf"  
+    value: "/egov-pdf" 
+  - name: PROJECT_DETAILS
+    value: "project-details"   
```

Comment:
@pramilaghosh-eGov  why details ?
It should be detail right and even key is exactly same in your app ?